packages/shared/src/features/onboarding/steps/FunnelPricing/FunnelPricingV2.tsx

```diff
@@ -0,0 +1,255 @@
+          startDate={discountStartDate}
+          isActive
+        />
+      )}
+      <PricingPlansV2
+        onChange={setSelectedPlan}
+        value={selectedPlan}
+        items={plans}
+      />
+      <Typography
+        className="text-center"
+        color={TypographyColor.Primary}
+        type={TypographyType.Callout}
+      >
+        {ctaMessage}
+      </Typography>
+      <Button
+        variant={ButtonVariant.Primary}
+        size={ButtonSize.Large}
+        color={ButtonColor.Cabbage}
+        className="w-full"
+        data-testid="funnel-pricing-cta"
+        onClick={() => onProceedToCheckout?.(selectedPlan)}
+      >
+        {cta || 'Get my plan'}
+      </Button>
+      <CreditCards>
+        <Typography
+          className="mb-4 flex gap-1 rounded-8 bg-surface-float px-2 py-0.5"
+          type={TypographyType.Callout}
+        >
+          <ShieldCheckIcon aria-hidden />
+          Pay safe & secure
+        </Typography>
+      </CreditCards>
+    </section>
+  );
+};
+
+const Pricing = ({
+  onTransition,
+  isActive,
+  parameters,
+}: FunnelStepPricingV2): ReactElement => {
+  const id = useId();
+  const { hero, features, plansBlock, discount, faq, trust, refund, reviews } =
+    parameters;
+  const [applyDiscount, setApplyDiscount] = useAtom(applyDiscountAtom);
+  const [discountStartDate, setTimer] = useAtom(discountTimerAtom);
+
+  useEffect(() => {
+    if (isActive && !discountStartDate) {
+      setTimer(new Date());
+    }
+  }, [isActive, discountStartDate, setTimer]);
+
+  const onProceedToCheckout = useCallback(
+    (plan) => {
+      onTransition({
+        type: FunnelStepTransitionType.Complete,
+        details: { plan, applyDiscount },
+      });
+    },
+    [onTransition, applyDiscount],
+  );
+
+  return (
+    <>
+      {/* Add a new style variant with CTA  */}
+      {!!discountStartDate && applyDiscount && (
+        <DiscountTimer
+          className="bg-brand-default text-white"
+          discountMessage={discount.message}
+          durationInMinutes={discount.duration}
+          isActive
+          onTimerEnd={() => setApplyDiscount(false)}
+          startDate={discountStartDate}
+          variant={DiscountTimerVariant.WithSlot}
+        >
+          <Button
+            className=" bg-white text-black"
+            href={`#${id}-plans`}
+            size={ButtonSize.Medium}
+            tag="a"
+            variant={ButtonVariant.Float}
+          >
+            {plansBlock.cta}
+          </Button>
+        </DiscountTimer>
+      )}
+      <div className="flex flex-col gap-6 px-4 py-6">
+        {/* Hero */}
+        {hero.image && (
+          <>
+            <Head>
+              <link rel="preload" as="image" href={hero.image} />
+            </Head>
+            <LazyImage
+              aria-hidden
+              eager
+              imgSrc={hero.image}
+              className="h-auto w-full object-center"
+              fit="contain"
+              ratio="64%"
```

Comment:
I think tailwind also supports some ratio, but not sure what this one is?

Reply:
I don't like it too, but removing this prop will make the LazyImageComponent disappear :(